plugins/woocommerce/src/StoreApi/Utilities/CartController.php

```diff
@@ -1365,11 +1365,17 @@ protected function sanitize_variation_data( $variation_data, $variable_product_a
 			if ( ! $attribute['is_variation'] ) {
 				continue;
 			}
+
 			$attribute_label          = wc_attribute_label( $attribute['name'] );
+			$lowercase_attribute_label = strtolower( $attribute_label );
```

Comment:
Not that I am entirely opposed to adding the usage of lowercase variation attribute names, is there a specific reason you added this?
For example is the actual label not available on the client side?

Reply:
![image](https://github.com/user-attachments/assets/cc700576-eb30-45e5-9011-38234300f8d7)

I did it this way as I was trying to get this to work with the existing add to cart form which has the input name in lower case. I am dabbling with trying to leverage this thunk and iAPI to do async add to cart ([and ultimately a same page checkout pattern](https://www.loom.com/share/016b138fbb1e4141b3b0047a4810456a))

Reply:
Gotcha, that makes sense, thanks for the clarification. Than I am ok with adding this logic.
There could be some small lint errors, you may want to align the equal signs ( `=` ) in the lines above.

And there is one additional space on line 1377 before the question mark: `isset( $variation_data[ $attribute_label ] )  ?`